npm-shrinkwrap.json

```diff
@@ -2725,7 +2725,7 @@
     },
     "ansi-escapes": {
       "version": "3.1.0",
-      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
```

Comment:
Shrinkwrap changes could be removed this PR since there are no changes to `package.json` neither.